lib/thredded/html_pipeline/onebox_filter.rb

```diff
@@ -74,6 +74,9 @@ def render_onebox(url)
         else
           preview.to_s.strip
         end
+      rescue => e
```

Comment:
This implicitly rescues `StandardError`. Perhaps doing it explicitly will fix the CodeClimate / Rubocop warning:

```ruby
rescue StandardError => e
```

Otherwise we can just `# rubocop:disable` it.